### PR TITLE
Add Kollaborate Marker Import 2.1.2.0

### PR DESCRIPTION
**Pull request by NaanProphet**
Imports Kollaborate marker lists into Final Cut Pro 7, FCP X
and Adobe Premiere.

### CODE REVIEW

File: Casks/kollaborate-marker-import.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+cask 'kollaborate-marker-import' do
+  version '2.1.2.0'
+  sha256 '82654c0b409af3ce609840e196c71989aa7a5c477488c9c99e773f01b8462917'
+
+  # digitalrebellion.com is the official download host per the vendor homepage
+  url "http://www.digitalrebellion.com/download/markerimport?version=#{version.no_dots}"
+  name 'Kollaborate Marker Import'
+  homepage 'https://www.kollaborate.tv/resources'
+  license :commercial
```
Comment (jawshooah) [inline Casks/kollaborate-marker-import.rb]: It's listed as free on the homepage. Why `:commercial`?

**Comment by SirDap** [comment]
My thinking was to follow convention as the [Backblaze](https://github.com/caskroom/homebrew-cask/blob/77b529d0011fcc687852b04a5c4bb1c1edf10433/Casks/backblaze.rb) cask: the application, while free, depends on and is useless without the commercial Kollaborate server.

**Comment by jawshooah** [comment]
> the application, while free, depends on and is useless without the commercial Kollaborate server

If the app itself is free to use, it should be `:gratis`. The distinction is important, as both free and commercial services often offer commercial apps.

**Comment by SirDap** [comment]
Thanks for the clarification! Read the documentation but am still learning the ropes.

**Comment by adidalal** [comment]
Thank you for the contribution. It needed some fixes, so they were made in a9b80bf24dd5bf8305cb1f5dc5466776cefe5428. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**